packages/app/src/runs/modals/CreateCloudOrgModal.vue

```diff
@@ -123,7 +124,7 @@ function startPolling () {
   setTimeout(() => {
     pause()
     polling.value = false
-  }, 180000)
+  }, 60000)
```

Comment:
Does the polling continue if the modal is dismissed? Should we be pausing/clearing the interval when this component is unmounted?

Reply:
We should... I thought I did TBH.

Reply:
yeah, if the modal is closed it stops polling